### PR TITLE
Small layout and style issues

### PR DESCRIPTION
**Pull request by Solcany**

### CODE REVIEW

File: components/TheExitConfirmation.vue [hunk 1]
```diff
@@ -1,20 +1,15 @@
...
-      <div class="lh-copy">
-        Are you sure you want to leave? <br>
-        All your progress will be lost.
-      </div>
-      <div class="mv3">
-        <ButtonPrimary @buttonClicked="onStay">Stay</ButtonPrimary>
-        <ButtonSecondary @buttonClicked="onLeave">Leave</ButtonSecondary>
-      </div>
-    </ModalTextRevealer>
+  <div v-show="isIdle" class="confirm-exit-container">
```
Comment (phivk) [inline components/TheExitConfirmation.vue:2]: what is `v-show="isIdle"` doing here?

File: plugins/idle-vue.js [hunk 2]
```diff
@@ -5,5 +5,5 @@ const eventsHub = new Vue()
 
 Vue.use(IdleVue, {
   eventEmitter: eventsHub,
-  idleTime: 60000,
+  idleTime: 300000,
```
Comment (phivk) [inline plugins/idle-vue.js:8]: idleTime should be 60000 == 60 seconds

**Review by phivk — CHANGES_REQUESTED** [review]
really great improvements to the look and feel, these make it feel a lot more whole ❤️ 

besides my inline comments, please see:

CardItem get's displayed under spectrum:
Screen Shot 2022-05-30 at 00 10 44

Buttons should have `cursor: pointer` to signal they are clickable:
Screenshot 2022-05-30 at 00 14 59

**Comment by Solcany** [comment]
@phivk 

> CardItem get's displayed under spectrum:

CardItem displays correctly on my end, but i increased z index of the draggable so please try again


> Buttons should have `cursor: pointer` to signal they are clickable:

They do have and display pointer on hover on my end

**Review by phivk — APPROVED** [review]

**Pull request merged**